resources/views/overons.blade.php

```diff
@@ -58,7 +58,7 @@
                     </div>
                     <div class="overons-text">
                         <h5>Jojo</h5>
-                        <div class="paragraph" style="text-align: left;">
+                        <div style="text-align: left;">
                             Wil de maatschappij laten zien, dat talenten van mensen met een handicap omarmend moet
                             worden en niet ontkend.
```

Comment:
Tekst zit in een div in plaats van een paragraaf, dit is wel belangrijk zeker voor de screenreaders